packages/debian/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+
+set -e
+set -u
+set -o pipefail
+
+
+# Build a DEB from the source files, using debuild.
+builddeb() {
+    set -x
+
+    mkdir -p /debbuild/build-pkg/src
+    cp -r /debbuild/pkg-meta/* /debbuild/build-pkg
+    cp -r /debbuild/pkg-src/* /debbuild/build-pkg/src
+
+    # Overwrite version if supported
+    sed -i 's/_VERSION_/'"${VERSION}"'/g' "/debbuild/build-pkg/debian/changelog"
+
+    pushd /debbuild/build-pkg 1>/dev/null
+    debuild -b -uc -us
+
+    cp -r /debbuild/*.deb /debbuild/result
+    chown -R "${TARGET_UID}:${TARGET_GID}" /debbuild/result
+}
+
+
+# Build a DEB from an RPM file, using alien.
+rpm2deb() {
+    set -x
+
+    alien --bump=0 /rpmbuild/source.rpm
+
+    cp /*.deb /debbuild/result
+    chown -R "${TARGET_UID}:${TARGET_GID}" /debbuild/result
+}
+
+
+case ${1:-''} in
```

Comment:
Details: Useless quotes, but it does not hurt, I think. :)

Reply:
Change for Debian and RedHat entrypoint :)